uber/templates/registration/feed.html

```diff
@@ -44,11 +44,11 @@ <h2>Feed of Database Actions</h2>
 </tr></thead>
 {% for tracked in feed %}
     <tr>
-        <td width="10%" valign="top" style="white-space:nowrap">{{ tracked.when|full_datetime_local }}</td>
-        <td width="5%" valign="top" style="white-space:nowrap">{{ tracked.who }}</td>
-        <td width="5%" valign="top" style="white-space:nowrap">{{ tracked.model }}</td>
-        <td width="25%" valign="top" style="white-space:nowrap">{{ tracked.which }}</td>
-        <td width="10%" valign="top" style="white-space:nowrap">
+        <td valign="top" style="white-space:nowrap; width=10%">{{ tracked.when|full_datetime_local }}</td>
```

Comment:
Last thing to fix! In the `style` attribute, you have to use a colon `:` instead of an equals `=`. Like
```
white-space:nowrap; width:10%
```